### PR TITLE
some BIG changes

### PR DESCRIPTION
**Pull request by OguzhanUmutlu**
lol

### CODE REVIEW

**Comment by Endermanbugzjfc** [comment]
Why remove `declare(strict_types=1);`

**Comment by Endermanbugzjfc** [comment]
@OguzhanUmutlu

**Comment by OguzhanUmutlu** [comment]
> Why remove `declare(strict_types=1);`

why not kekw

**Comment by Endermanbugzjfc** [comment]
> > Why remove `declare(strict_types=1);`
> 
> why not kekw

do you even know what it does

**Comment by OguzhanUmutlu** [comment]
> Why remove `declare(strict_types=1);`

why not kekw

**Comment by Endermanbugzjfc** [comment]
you should remove it when the author already added it in their code

File: src/MrNinja008/ItemInteract/Main.php [hunk 1]
```diff
@@ -1,76 +1,47 @@
 <?php
 
-declare(strict_types=1);
```
Comment (Thunder33345) [inline src/MrNinja008/ItemInteract/Main.php:3]: bad

**Review by Thunder33345 — CHANGES_REQUESTED** [review]
removed stricts type, need fixing

**Comment by Endermanbugzjfc** [comment]
lmao even thunder joined to this shit posting convo

**Pull request closed without merging**